server/typedefs/Jobs.js

```diff
@@ -0,0 +1,25 @@
+const { gql } = require('apollo-server-express');
+
+const jobTypeDefs = gql`
+    type Job {
+        _id: ID!
+        title: String!
+        description: String
+        dateRequested: String!
+        homeowner: User
+        gardener: User
+        plot: Plot
+        status: String
+    }
+    type Query {
+        jobs: [Job]
+        job(jobId: ID!): Job
```

Comment:
I think is supposed to be
```suggestion
        job(id: ID!) : Job
```